bindata/network/ovn-kubernetes/ovnkube-node.yaml

```diff
@@ -58,13 +66,19 @@ spec:
           readOnly: true
         - mountPath: /etc/openvswitch
           name: host-config-openvswitch
+        - mountPath: /var/run/ovn-config
+          name: ovn-config-mount
         resources:
           requests:
             cpu: 100m
             memory: 300Mi
         env:
         - name: OVN_DAEMONSET_VERSION
-          value: "3"
+          value: "4"
+        lifecycle:
+          preStop:
+            exec:
+              command: ["/root/ovnkube.sh", "cleanup-ovs-server"]
```

Comment:
@pecameron I looked at the upstream cleanup stuff for a while today RE our question Monday about cleanup. And I think we just don't follow upstream here at all, we just leave everything be. Reason is because we'll pick things up and reconcile on restart, and we may restart often for upgrades.

Reply:
OK